validators.py

```diff
@@ -107,3 +107,22 @@ def validate(expected_capacity, **kwargs):
                 f'CE Unit Expected Capacity "{expected_capacity}" cannot be null, blank or zero')
 
     return validate
+
```

Comment:
I don't think you should leave a massive comment block in the code

Reply:
Good spot